jha/src/settings.js

```diff
@@ -1,4 +1,4 @@
 export default {
   bypassRouteGuards: false,
-  useDummyData: false,
+  useDummyData: true,
```

Comment:
Please have this set to `false`. This is only for development testing purposes.

Reply:
updated, sorry I forgot to reset it